go/internal/indexerhandler/handle.go

```diff
@@ -296,8 +296,19 @@ func (h *Handler) handleExecute(e *Message) error {
 				return errors.Wrap(err, "failed to handle create album")
 			}
 		}
+	case "add_to_library": //Add user's album to my library
+		if executeMsg.Contract == h.config.Network.MusicplayerContractAddress {
```

Comment:
put this check in handler so it can be reused
(for example by the dao exec handler)

Reply:
@n0izn0iz 
I think, above code is correct.